chapters/installation-setup.qmd

```diff
@@ -24,12 +24,12 @@ author: ""
 ## Acknowledgements
 
 ```{r}
-#| eval: false
+#| eval: true
 #| echo: false
 #| message: false
 #| warning: false
 #| output: asis
-bibtexkeys = c("koziar2023", "bryan2023")
```

Comment:
@konradpa, do you know why `koziar2023` is removed here?

Reply:
nope, readded